src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeløpsperiodeBarnetilsynDto.kt

```diff
@@ -1,11 +1,17 @@
 package no.nav.familie.ef.sak.beregning.barnetilsyn
 
-import no.nav.familie.ef.sak.felles.dto.Periode
+import no.nav.familie.kontrakter.felles.Datoperiode
+import no.nav.familie.kontrakter.felles.Periode
 import java.math.BigDecimal
 import java.util.UUID
 
 data class BeløpsperiodeBarnetilsynDto(
-    val periode: Periode,
+    val fellesperiode: Datoperiode,
```

Comment:
Hvis man kaller denne periode, og den andre deprecatedPeriode, samt bruker JsonProperty for å beholde de tidligere navnen, så trenger man kanskje ikke all refakturering av kode i backend? 

For å unngå alle "fellesperiode", men beholde periode i backend.

Reply:
Periode skulle vel også kunnet blitt flyttet ned til 
```data class ...(
...
) {
@Deprecated("Bruk fellesperiode", ReplaceWith("fellesperiode"))
    val periode: no.nav.familie.ef.sak.felles.dto.Periode = no.nav.familie.ef.sak.felles.dto.Periode(
        fellesperiode.fom,
        fellesperiode.tom
    ),
}
```